parent/pom.xml

```diff
@@ -844,6 +833,33 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>${maven.compiler.plugin.version}</version>
+                    <configuration>
+                        <compilerArgs>
+                            <arg>-XDcompilePolicy=simple</arg>
+                            <arg>
+                                -Xplugin:ErrorProne
+                                -XepDisableWarningsInGeneratedCode
+                                -XepExcludedPaths:(.*/src/integrationTest/.*|.*/src/test/.*|.*/modules/(raft|metastorage-server|schema|table)/.*)
```

Comment:
I guess we have separate issues to fix code in these modules, right?

Reply:
Yes, I''ll create tickets.